src/app/room/components/room.component.scss

```diff
@@ -28,4 +28,11 @@ button {
 .invisible {
   opacity: 0;
   pointer-events: none;
-}
\ No newline at end of file
+}
+
+.text-container {
+  display: flex;
+  justify-content: center;
+  font-family: emoji;
+  font-size: medium;
```

Comment:
What do you think about changing the font in the root for everything then? We don't have much text (yet?), but I don't like the idea of it being different in different places for no reason.

Reply:
I changed on the root level, but I meant here to change only for this title